pkg/config/config.go

```diff
@@ -86,6 +86,12 @@ func New(v *viper.Viper, co *Options) (*Config, error) {
 				} else {
 					return nil, fmt.Errorf("%s cannot be empty when using Kubernetes Auth", types.EnvAvpK8sRole)
 				}
+			case types.TokenAuth:
+				if v.IsSet(types.EnvAvpVaultToken) {
```

Comment:
You should be able to use the vault const here since its exported `api.EnvVaultToken` https://github.com/hashicorp/vault/blob/main/api/client.go#L42 (I think)